backend/handlers/get_rules.go

```diff
@@ -11,8 +11,30 @@ import (
 
 func GetRules(postgresDb *gorm.DB) func(w http.ResponseWriter, r *http.Request) {
```

Comment:
should this just mirror GetComplianceFramework in terms of handling arguments to api call